### ISSUE TITLE
Data corruption issues 

### ISSUE THREAD

**Issue by ridhaalkhabaz** [msaveski/toxic_conversation_structure#2]
Hi, 

I hope you are doing well. I am here to see if someone else is facing the same issue. I tried downloading the data and unzipping it using The Unarchiver and other applications. However, I was not successful with any tool, it was always prompting me that the data might be incomplete.

**Comment by msaveski** [comment]
Hi, thanks for the note! What software and OS are you using? I would try the official 7zip tools, following the instructions from the readme. Below is the relevant snippet.

To uncompress the data run the following [7zip](https://www.7-zip.org/) commands:
```
7za x news_jsons.7z
7za x midterms_jsons.7z
7za x news_jsons_no_embs.7z
7za x midterms_jsons_no_embs.7z
```

**Comment by ridhaalkhabaz** [comment]
Hi Martin,

Thank you for replying to my issue. 
I tried multiple software and OS. I could only uncompress `news_jsons_no_embs.7z.001` and `midterms_jsons_no_embs.7z.001`. For some reason, I get a writing error whenever I try to uncompress by any software on MacOS or using 7zip on Windows 11. Please let me know if you have a solution to get a copy of the data with embedding.

**Comment by msaveski** [comment]
Are you able to see how which file it gets stuck on?

**Comment by ridhaalkhabaz** [comment]
It was different for each file. Let me try again and get you more details.

**Comment by ridhaalkhabaz** [comment]
Hi Martin, 

Thank you for being patient with me. 
I keep getting this error with any data file with embedding. 
![error](https://github.com/msaveski/toxic_conversation_structure/assets/46946471/168b90c5-93bd-41e5-a067-cf929213d4e1)

**Comment by msaveski** [comment]
I can't see the command you are running before you get the error.

Some things I would try:
- I think you shouldn't pass the ".001" extension
- Note that I'm using the 7za command; it seems like there is some
difference between 7z and 7za
- Compare the md5 hashes of the downloaded files with those in the
dataverse.

Are you able to extract the news_jsons data?
